web/app/remark.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+/** @jsx h */
+import loadPolyfills from '@app/common/polyfills';
+import { h, render } from 'preact';
+import 'preact/debug';
+
+import { Provider } from 'preact-redux';
+import { ConnectedRoot } from '@app/components/root';
+import { UserInfo } from '@app/components/user-info';
+import reduxStore from '@app/store';
+
+// importing css
+import '@app/components/list-comments';
+
+import { NODE_ID } from '@app/common/constants';
+import { StaticStore } from '@app/common/static_store';
+import api from '@app/common/api';
+
+if (document.readyState === 'loading') {
+  document.addEventListener('DOMContentLoaded', init);
+} else {
+  init();
+}
+
+async function init(): Promise<void> {
+  await loadPolyfills();
+
+  const node = document.getElementById(NODE_ID);
+
+  if (!node) {
+    console.error("Remark42: Can't find root node.");
+    return;
+  }
+
+  const params = window.location.search
+    .replace(/^\?/, '')
+    .split('&')
+    .reduce(
+      (memo, value) => {
+        const vals = value.split('=');
+        if (vals.length === 2) {
+          memo[vals[0]] = vals[1];
+        }
+        return memo;
+      },
+      {} as any
+    );
+
+  StaticStore.config = await api.getConfig();
+
+  if (params.page === 'user-info') {
+    render(
+      <div id={NODE_ID}>
+        <div className="root root_user-info">
+          <Provider store={reduxStore}>
+            <UserInfo
+            // wtf?
```

Comment:
what happens here?

Reply:
Oh, thanks for pointing out. It was my point of confusion: there were function `onClose` decraled https://github.com/umputun/remark/blob/084a7a9f992d2e52b9d33dc2df5d57b67aa71a5b/web/app/remark.js#L53-L58

but then it seems it not used anywhere in [UserInfo](https://github.com/umputun/remark/blob/084a7a9f992d2e52b9d33dc2df5d57b67aa71a5b/web/app/components/user-info/user-info.jsx#L1), so I left comment (should have marked it as todo:) to check out later, then checked that it's indeed dead code, removed onClose, but forgot to remove comment. It should be removed